arduino/OpenMRN.h

```diff
@@ -465,25 +476,36 @@ class OpenMRN : private Executable
     }
 
 #if defined(ESP32)
+
+    /// Creates a FreeRTOS task on the ESP32 to automatically call the @ref loop
+    /// method. If the ESP32 hardware CAN device is in use the background task
+    /// is strongly suggested to be used. @ref start_background_task.
     static void openmrn_background_task(void *param)
     {
+        /// Get handle to our parent OpenMRN object for the loop method access
         OpenMRN *openmrn = reinterpret_cast<OpenMRN *>(param);
-        while(true)
+
+        // Add this task to the WDT, NULL is passed in to have the
+        // WDT code find the task handle automatically
+        esp_task_wdt_add(NULL);
+
+        while (true)
         {
-            // Feed the watchdog so it doesn't reset the ESP32
-            esp_task_wdt_reset();
+            // it is not necessary to feed the WDT here as it is done internally
```

Comment:
i think this comment can be safely removed.

Reply:
removed

Reply:
removed